site/pages/forgot-password.page.tsx

```diff
@@ -0,0 +1,62 @@
+import { NextPageContext } from "next";
+import { parseCookies } from "nookies";
+import { ReactElement, useState } from "react";
+import ErrorSummary from "../components/ErrorSummary";
+import CsrfForm from "../components/form/CsrfForm";
+import TextInput from "../components/form/TextInput";
+import { BaseLayout } from "../components/layout/Layout";
+import { COOKIES_FORGOT_PASSWORD_ERRORS } from "../constants";
+import { PageState } from "../interfaces";
+import { ForgotPasswordSchema, forgotPasswordSchema } from "../schemas/forgot-password.schema";
+import { destroyCookieOnResponseObject, getPageState } from "../utils/apiUtils";
+import { getStateUpdater } from "../utils/formUtils";
+
+const title = "Reset password - Create Transport Disruptions Service";
+const description = "Password reset page for the Create Transport Disruptions Service";
+
+export interface ForgotPasswordPageProps extends PageState<Partial<ForgotPasswordSchema>> {}
+const ForgotPassword = (props: ForgotPasswordPageProps): ReactElement => {
+    const [pageState, setPageState] = useState(props);
+    const stateUpdater = getStateUpdater(setPageState, pageState);
+    return (
+        <BaseLayout title={title} description={description}>
+            <h1 className="govuk-heading-xl">Forgot your password?</h1>
+            <div className="govuk-grid-row">
+                <div className="govuk-grid-column-two-thirds">
+                    <>
+                        <CsrfForm action="/api/forgot-password" method="post" csrfToken={pageState.csrfToken}>
+                            <ErrorSummary errors={pageState.errors} />
```

Comment:
When I enter an email with no account linked to it - it says reset password sent (do we want that to happen or do we want to send an error back?)

Reply:
Yep, that's the intended functionality. Double checked with Chris and this matches BODS process.